src/drivers/intel/cavs/ipc.c

```diff
@@ -23,6 +23,35 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#if CAVS_VERSION >= CAVS_VERSION_1_8
+
+/** \brief Bit set by host to indicate msg payload is in regs */
+#define IPC_DIPCTDR_MSG_REG		BIT(30)
```

Comment:
Missing #include <sof/bit.h>.

Reply:
Inlcuded in the ipc/header-intel-cavs.h as all defs moved there.